src/main/resources/application.properties

```diff
@@ -11,3 +11,8 @@ spring.jpa.properties.hibernate.format_sql=true
 spring.autoconfigure.exclude[0]=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
 
 #spring.liquibase.drop-first=true
+server.port=9000
+
+# swagger-ui custom path
+springdoc.api-docs.enabled=true
+springdoc.swagger-ui.path=/swagger-ui.html
```

Comment:
is this not the default path?

Reply:
yes. but added there explicitly still to address that can create custom swagger ui URL

Reply:
I can't foresee a case where we would need to change swagger's URL